client/my-sites/plugins/plugins-list/index.tsx

```diff
@@ -1,8 +1,7 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { WPCOM_FEATURES_MANAGE_PLUGINS } from '@automattic/calypso-products';
-import { localize } from 'i18n-calypso';
+import { localize, translate } from 'i18n-calypso';
```

Comment:
I wonder if we can just import the type if the only reason to have the import is to do a `typeof`? `i18n-calypso` comes with its own types, so we can just use them.

```
import type { translate } from 'i18n-calypso';
```

Reply:
I couldn't get `translate` to work as it seems to be declared only as a value. It did have the type specified in `I18N` though, so I used that.